toolchain/check/testdata/function/declaration/no_prelude/name_poisoning.carbon

```diff
@@ -311,10 +311,21 @@ interface A {
 class X {
   extend impl as A {
     fn F() { return; }
```

Comment:
Hopefully we can reduce the noise by changing this.
```suggestion
    fn B() { }
```

I'm surprised it wasn't complaining that `F` is not a member of `A`. Perhaps that should be a todo_fail test.

Reply:
Yes, this does reduce noise. But also this test is not really testing poisoning since we just reject the `impl as` outright. I will move this test over to the /impl/ tests.

And added a todo test.